### PR TITLE
build amzn2 packages

### PR DESCRIPTION
**Pull request by id**
Fixes [EMQX-8449](https://emqx.atlassian.net/browse/EMQX-8449)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/` dir
- [ ] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

File: changes/v5.0.12-zh.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ v5.0.11 或更早版本创建的配置文件，在新版本中会被自动转换
   在此之前，环境变量的重载仅作用于已经存在的配置之上，例如，当 `emqx.conf` 中已经配置了一个 MySQL 认证，那么可以使用如下方法来将它禁用：
   `env EMQX_AUTHENTICATION__1__ENABLE=false`。
 
+- Start building packages for Amazon Linux 2 [#9537](https://github.com/emqx/emqx/pull/9537).
```
Comment (id) [inline changes/v5.0.12-zh.md:44]: @zmstone need translation

Comment (zmstone) [inline changes/v5.0.12-zh.md:44]:
```suggestion
- 为 Amazon Linux 2 平台发布安装包 [#9537](https://github.com/emqx/emqx/pull/9537)。
```

**Review by zmstone — APPROVED** [review]

File: .github/workflows/build_slim_packages.yaml [hunk 2]
```diff
@@ -29,17 +29,14 @@ jobs:
       fail-fast: false
       matrix:
         profile:
-        - emqx
-        - emqx-enterprise
+        - ["emqx", "el7"]
+        - ["emqx-enterprise", "ubuntu20.04"]
```
Comment (id) [inline .github/workflows/build_slim_packages.yaml:33]: @zmstone missed your comment in https://github.com/emqx/emqx/pull/9497
do we need to swap these? I.e. build emqx-enteprise-el7 and emqx-ubuntu-20.04 instead?

Comment (zmstone) [reply .github/workflows/build_slim_packages.yaml:33]: It’s ok

**Pull request merged**